Documentation/ceph-csi-drivers.md

```diff
@@ -134,15 +138,15 @@ metadata:
   name: csi-rbd-secret
   namespace: default 
```

Comment:
The secret should be in the `rook-ceph` namespace. This is where all the cluster secrets are stored. I assume this means the csi properties above in the storage class should also point to the `rook-ceph` namespace.

Reply:
Looks like we need another step to create the rbd pool specified in the storage class. Rook doesn't create it by default.

Reply:
added